### PR TITLE
Diversity/Supporter release

### PR DESCRIPTION
**Pull request by Hoverbear**
From https://github.com/RustFestEU/blog.rustfest.eu/pull/23

Please have a look on the post. It's a bit lenghty, so please help me edit it for brevity and understandibility.

The links I'll have to update with the final links before this can go live, but besides that it's good enough. :)

(Side note: Please just make branches on the `rustfest` repo so we can all push!)

### CODE REVIEW

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Supporter Tickets on-sale, Diversity Ticket applications now open"
+socialImageSrc: '/assets/posts/diversityticketsorg.png'
+authors:
+  - flaki
+  - hoverbear
+---
+
+From now on you can buy [Supporter Tickets to RustFest Zurich](https://ti.to/asquera-event-ug/rustfest-zurich)! Secure your place and make the conference more awesome!
```
Comment (dns2utf8) [inline _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: The link is not public yet

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## Travel & accommodation support for awardees
+Besides free conference tickets we are able to offer support for expenses related to the conference (including travel, accommodation, visa fees, etc.) to applicants who request these. Please let us know in your application what kind of support would you need to attend RustFest, and we'll try our best to help you. Feel free to e-mail us with any questions, and check the announcement at [diversitytickets.org](https://diversitytickets.org/events/100) for details.
+
+
+## Diversity support pool donations
+All 15 diversity tickets offered at RustFest are already budgeted and are offered to eligible applicants. We understand, however, that just offering free tickets sometimes isn't sufficient for someone to afford and be able to attend. For this reason, we try to provide every assistance (documentation, help with visas, childcare (see below) etc.) and monetary (travel, accommodation support, visa fees etc.). As with every organization out there, our time and resources are equally limited. To offset this, we have created something called a "Diversity Sponsorship pool".
```
Comment (dns2utf8) [inline _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: Can we make the (see below) clickable?

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Our Diversity Sponsorship pool is a monetary fund reserved for the sole purpose of enabling our diversity program's awardees to attend and enjoy RustFest. Its initial budget is seeded by our sponsors and various other revenue streams (e.g. ticket sales), for Zurich this budget is 3,000 Euros. That, as you might imagine is very slim to be shared by 15 awardees, which is the point where you can help!
+Today, alongside our Supporter tickets (see below) we're opening our [Diversity Sponsorship pool for donations](https://ti.to/asquera-event-ug/rustfest-zurich). These are pay-what-you-want style donations that you can find in the RustFest ti.to ticket shop, and by buying them you support us in our mission to help more of our awardees afford enjoying RustFest with us.
+
+> Please note that these "tickets" do not include conference access!
+> A separate conference ticket need to be purchased for attending RustFest.
```
Comment (dns2utf8) [inline _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: need -> needs

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+# Supporter conference tickets
+In a so-called "reverse early bird" fashion we are also releasing today our first batch of tickets that are called the Supporter Conference tickets. At RustFest we try to keep ticket prices to a minimum to make sure a wide range of people can afford buying them. This, as you might imagine limits our possibilities budget-wise, and to be able to provide (and afford) some non-essential services that would enhance our attendees' experience, we had to make hard decisions. Some of these deemed "non-essential" services include talk recordings, live talk transcription, on-site childcare and conference T-shirts, in no particular order. Please note that non-essential here doesn't mean we don't think these features are important, but rather the fact that it is possible to have a conference without talk recordings -- but e.g. not without a venue.
+
+We welcome [company sponsorships](http://zurich.rustfest.eu/sponsoring/), but by buying the [Supporter tickets](https://ti.to/asquera-event-ug/rustfest-zurich) anyone can contribute to these goals.
```
Comment (dns2utf8) [inline _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: The sponsoring page looks out of date

**Review by dns2utf8 — CHANGES_REQUESTED** [review]
After reading the whole thing I am confused.
The supporter tickets are describes as two things:

1. to help others, but not go to the conf with that ticket
2. to go to the conf with the ticket and help others.

Can we clarify this?

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md (hunk 4 above)
Comment (Hoverbear) [reply _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: Oh no

**Comment by Hoverbear** [comment]
@dns2utf8 Does that help clarify it a bit?

File: _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md (hunk 4 above)
Comment (dns2utf8) [reply _posts/2017-08-03-diversity-sponsorship-supporter-tickets-live.md]: this is fixed now

**Review by dns2utf8 — APPROVED** [review]

**Comment by dns2utf8** [comment]
LGTM 👍

**Pull request merged**